Exiled.Events/Patches/Generic/InventoryControlPatch.cs

```diff
@@ -77,7 +77,8 @@ private static void AddItem(Player player, ItemBase itemBase, ItemPickupBase ite
         {
             Item item = Item.Get(itemBase);
 
-            Pickup.Get(itemPickupBase)?.GetPickupInfo(item);
+            if (itemPickupBase != null)
+                item.Scale = itemPickupBase.transform.localScale;
```

Comment:
Thats a strange fix, as i see you remove main logic for your pr: saving properties between items/pickups and projectiles

Not better will be 
```cs
if (ipb != null)
    Pickup.Get(ipb).GetPickupInfo(item)
```
?

Or i misunderstand something?

Reply:
oh wait yes